### PR TITLE
refactor!: only support winston3

### PR DESCRIPTION
**Pull request by ofrobots**
BREAKING CHANGE: Drop support for winston2 as a semver major change.

Ref: https://github.com/googleapis/nodejs-logging-winston/pull/278

### CODE REVIEW

**Comment by ofrobots** [comment]
This unblocks #278. We may spin off winston2 support (if it is still necessary) into a separate package, or continue to support it from the penultimate semver major branch.

File: system-test/logging-winston.ts [hunk 1]
```diff
@@ -24,11 +24,9 @@ import {ErrorsApiTransport} from './errors-transport';
 const inject = require('require-inject');
 
 const winston3 = require('winston');
```
Comment (kjin) [inline system-test/logging-winston.ts]: Nit, call this `winston`?

Comment (ofrobots) [reply system-test/logging-winston.ts]: done.

**Comment by ofrobots** [comment]
@JustinBeckwith this is good to land, but an Administrator will have to force land this. Snyk is complaining about the removal of a package.json file. This is intentional.

**Pull request merged**